WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptEmail/ReceiptEmailView.swift

```diff
@@ -0,0 +1,116 @@
+import SwiftUI
+import Yosemite
+
+struct ReceiptEmailView: View {
```

Comment:
Any feedback is welcome on this view as well. I tried to keep it simple, maybe it's too simple.

Another option I considered is reusing 'customer form view', with the email field focused. However, I thought it may distract the user in this flow and left only the email field.

Reply:
I think is perfectly fine for our current use case, and we can iterate on the future depending on, for example:
- Are we going to be able to select an email from existing customers, rather than writing it?
- Are going to be able to "resend" to the same address (eg: retry button, reusing the already-entered email) and/or different addresses prompted from the order view?

Something I've missed is some sort of UI feedback between sending was completed and all the modals were dismissed, something like what we do when creating a coupon (a big animated checkmark before calling for dismiss)